app/views/publish/courses/confirmation.html.erb

```diff
@@ -79,7 +79,7 @@
           ) %>
         <% end %>
 
-        <% if @provider.accredited_provider? %>
+        <% if @provider.accredited_provider? && !course.recruitment_cycle_after_2023? %>
```

Comment:
Do we have a trello card to clean all of those checks after the recruitment cycle ends?

Reply:
I've added a reminder card here: https://trello.com/c/3KlvjCa0/334-rollover-code-cleanup